src/probnum/filtsmooth/gaussfiltsmooth/gaussfiltsmooth.py

```diff
@@ -45,7 +45,7 @@ def filtsmooth(self, dataset, times, **kwargs):
         """
         dataset, times = np.asarray(dataset), np.asarray(times)
         filter_posterior = self.filter(dataset, times, **kwargs)
-        smooth_posterior = self.smooth(filter_posterior, **kwargs)
+        smooth_posterior = self.smooth(filter_posterior)
```

Comment:
It was a mistake^^. But yeah, I agree. I dont dare removing all of them though, at least not now, because there are some subtle ones that are passed around (for instance the ominous `step` variable for the CKE)